R/qenv-within.R

```diff
@@ -1,34 +1,28 @@
-#' Evaluate expression in `qenv` object.
+#' Evaluate Expression in `qenv`
 #'
-#' Convenience function for evaluating inline code inside the environment of a `qenv`.
+#' @details
+#' `within` is a convenience function for evaluating inline code inside the environment of a `qenv`.
+#' It is a method for the `base` generic that wraps `eval_code` to provide a simplified way of passing code.
+#' `within` accepts only inline expressions (both simple and compound) and allows for injecting values into `expr`
+#' through the `...` argument:
+#' as `name:value` pairs are passed to `...`, `name` in `expr` will be replaced with `value`.
 #'
-#' This is a wrapper for `eval_code` that provides a simplified way of passing code for evaluation.
-#' It accepts only inline expressions (both simple and compound) and allows for injecting values into `expr`
-#' through the `...` argument: as `name:value` pairs are passed to `...`,
-#' `name` in `expr` will be replaced with `value`.
-#'
-#' @section Using language objects:
+#' @section Using language objects with `within`:
 #' Passing language objects to `expr` is generally not intended but can be achieved with `do.call`.
 #' Only single `expression`s will work and substitution is not available. See examples.
 #'
-#' @param data `qenv` object
-#' @param expr `expression` to evaluate
-#' @param ... `name:value` pairs to inject values into `expr`
+#' @param data (`qenv`)
+#' @param expr (`expression`) to evaluate. Must be inline code, see `Using language objects...`
+#' @param ... see `Details`
 #'
 #' @return
-#' Returns a `qenv` object with `expr` evaluated. If evaluation raises an error, a `qenv.error` is returned.
-#'
-#' @seealso [`eval_code`], [`base::within`]
+#' `within` returns a `qenv` object with `expr` evaluated or `qenv.error` if evaluation fails.
 #'
-#' @export
-#'
-#' @rdname within
+#' @seealso [`base::within`]
```

Comment:
If `qenv` docs is a "starting point" it is good to link other qenv methods
```suggestion
#' @seealso [`base::within`], [get_var()], [get_warnings()], [join()], [concat()]
```